WordPress/src/main/java/org/wordpress/android/ui/sitecreation/SiteCreationActivity.kt

```diff
@@ -131,6 +146,11 @@ class SiteCreationActivity : LocaleAwareActivity(),
         ActivityUtils.hideKeyboard(this)
```

Comment:
I'm wondering if we should use the feature / ExPlat flag(s) here to prevent keyboard bounce. :thinking: It is a minor issue:

* When the site name is OFF, site designs follow intents (so it makes sense to hide)
* When the site name is ON, site name follows intents (so we probably want to keep the keyboard visible)

Reply:
:+1: Added here: https://github.com/wordpress-mobile/WordPress-Android/pull/16259/commits/5f66466996890712c37cb22c0ea677256bc3cb47